test/test_networks/test_conditional_hint_network.jl

```diff
@@ -14,11 +14,16 @@ batchsize = 2
 L = 2
 K = 2
 
-# Multi-scale and single scale network
-CH0 = NetworkMultiScaleConditionalHINT(n_in, n_hidden, L, K; split_scales=false, k1=3, k2=1, p1=1, p2=0)
-CH1 = NetworkConditionalHINT(n_in, n_hidden, L*K; k1=3, k2=1, p1=1, p2=0)
+# single scale network
+CH0 = NetworkConditionalHINT(n_in, n_hidden, L*K; k1=3, k2=1, p1=1, p2=0)
+CH1 = NetworkConditionalHINT(n_in, n_hidden, L*K; logdet=false, k1=3, k2=1, p1=1, p2=0)
 
-nets = [CH0, CH1, reverse(CH1)]
+# Multi-scale network 
```

Comment:
Wrong file, there is a multiscae test.

Reply:
I didnt understand this comment, could you please elaborate?

Reply:
This test is in the wrong file, there is a different one for multiscale network.

Reply:
If you are referring to test_multiscale_hint_network.jl, that is for non-conditional multiscale. The only previous test I found for conditional multiscale was in this file that I am editing.

Reply:
then make a new file for it so it;'s easy to find

Reply:
Will do!